src/pages/basic_data/SearchForCristinPerson.tsx

```diff
@@ -42,13 +42,14 @@ export const SearchForCristinPerson = ({
   const [searchQuery, setSearchQuery] = useState('');
   const debouncedSearchQuery = useDebounce(searchQuery);
   const searchQueryIsNumber = !!searchQuery && !isNaN(Number(searchQuery));
+  const sortBy = 'score desc';
 
   const [isLoadingSearchByNin, setIsLoadingSearchByNin] = useState(false);
 
   const [searchByNameResponse, isLoadingSearchByName] = useFetch<SearchResponse<CristinPerson>>({
     url:
       searchQuery && searchQuery === debouncedSearchQuery && !searchQueryIsNumber
-        ? `${CristinApiPath.Person}?results=${searchSize}&name=${debouncedSearchQuery}`
+        ? `${CristinApiPath.Person}?results=${searchSize}&name=${debouncedSearchQuery}&sort=${sortBy}`
```

Comment:
siden det bare brukes her kunne det kanskje like godt ligget her også, og da slippe å lage en egen const når den kanskje ikke vil brukes heller?

```suggestion
        ? `${CristinApiPath.Person}?results=${searchSize}&name=${debouncedSearchQuery}&sort=score desc`
```